content/3_Silver/Prefix_Sums.mdx

```diff
@@ -272,12 +272,16 @@ public class Main {
 
 <PySection>
 
+In Python you can use [itertools.accumulate](https://docs.python.org/3.6/library/itertools.html#itertools.accumulate). Note that you have to call list() since accumulate returns a [generator](https://wiki.python.org/moin/Generators). The first value returned is that of the  iterable passed, so we add a zero in the front (in newer versions you can give the optional parameter of initial=0).
```

Comment:
3.6?

Reply:
The current python version is 3.6 for USACO if I am not mistaken. 3.12 is the latest.